content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -134,9 +134,9 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 ```
 
-Finally, a `ClusterRoleBinding` may be used to grant permission at the cluster level and in all
-namespaces. The following `ClusterRoleBinding` allows any user in the group "manager" to read 
-secrets in any namespace.
+To grant permissions across a whole cluster, you can use a ClusterRoleBinding.
+The following ClusterRoleBinding allows any user in the group "manager" to read
+Secrets in any namespace.
```

Comment:
prefer lowercase plural `secrets` as the resource, corresponding to what appears in the role